demo-web/src/lib.rs

```diff
@@ -0,0 +1,558 @@
+#[cfg(not(target_arch = "wasm32"))]
+mod main_native;
+#[cfg(not(target_arch = "wasm32"))]
+pub use main_native::*;
+
+#[cfg(target_arch = "wasm32")]
+mod main_web;
+#[cfg(target_arch = "wasm32")]
+pub use main_web::*;
+
+use rafx::api::{
```

Comment:
Do we not have a prelude for this?

Reply:
We don't have a prelude but ::* does make sense here.